src/briefcase/commands/package.py

```diff
@@ -6,6 +8,16 @@
 from .base import BaseCommand, full_options
 
 
+class DeprecatedAction(argparse.Action):
+    def __call__(option_strings, *args, **kwargs):
+        """Use warnings.warn to tell the user that this argument is deprecated and will
+        have no effect."""
+        warnings.warn(
+            f"The {option_strings} argument is no longer used, and has no effect.",
+            DeprecationWarning,
+        )
+
+
```

Comment:
I could easily move this somewhere central if we think it would be useful elsewhere. Alternatively, I'm open to other suggestions, including raising an error as opposed to just warning.

Reply:
Honestly - my inclination is to just remove it completely. APIs need to be very careful about backwards compatibility, but this is a command that will (mostly) be executed by humans - they're a lot more adaptable.

Reply:
Good news. Because I actually did this very wrong anyway.... 😅